.devcontainer/Dockerfile.base

```diff
@@ -0,0 +1,42 @@
+FROM runpod/pytorch:2.4.0-py3.11-cuda12.4.1-devel-ubuntu22.04
```

Comment:
Might be worth checking later how big is this image compared to a raw nvidia+installing python+pytorch manually.

Reply:
Saved about 8 gigs by switching to `nvidia/cuda:12.2.2-cudnn8-devel-ubuntu22.04`